recipe/meta.yaml

```diff
@@ -24,34 +24,31 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake
     - make   # [unix]
-    - {{ cdt('mesa-libGL-devel') }}  # [linux]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
     - {{ cdt('libselinux') }}        # [linux]
     - {{ cdt('libxcb') }}            # [linux]
     - {{ cdt('libxdamage') }}        # [linux]
-    - {{ cdt('libxfixes') }}         # [linux]
     - {{ cdt('libxxf86vm') }}        # [linux]
-    - {{ cdt('libxext') }}           # [linux]
     - {{ cdt('libxau') }}            # [linux]
-    - {{ cdt('mesa-libGL-devel') }}  # [linux]
     - {{ cdt('expat') }}             # [linux]
     - {{ cdt('mesa-dri-drivers') }}  # [linux]
     - freetype                       # [linux]
     - gnuconfig                      # [unix]
+    - xorg-libxext
+    - xorg-libxrender
+    - xorg-libxfixes
+    - libglu    # [linux]
+    - jpeg
+    - libpng
```

Comment:
These should be host dependencies.

Reply:
Right, but for some reason putting them into `build:` fixes many of the linking issues when building FLTK with shared libraries. I'm not sure what is going on there. Maybe some kind of conflict between the X11-related  yum/cdt/conda packages?

Reply:
Turns out fltk ignores LDFLAGS for shared libraries, it uses DSOFLAGS instead.